Tests/SonarScanner.MSBuild.PreProcessor.Test/SonarWebServerTest.cs

```diff
@@ -39,22 +39,19 @@ public class SonarWebServerTest
 {
     private const string ProjectKey = "project-key";
 
-    private readonly IDownloader downloader;
-    private readonly TestLogger logger;
-    private readonly Version version;
+    private readonly TestLogger logger = new();
+    private IDownloader downloader;
+    private Version version;
 
     private SonarWebServerStub sut;
 
-    public SonarWebServerTest()
+    [TestInitialize]
+    public void Init()
     {
-        downloader = Substitute.For<IDownloader>();
         version = new Version("9.9");
-        logger = new TestLogger();
```

Comment:
This file is way too big, so I made the initialization in `TestInitialize` instead of the constructor.

The `SQ` and `SC` server tests are initializing the downloader in-line.

I might come back to this file and make everything in-line, but I would like to merge this and finish the resolver today/tommorow, so that we can start ITs and manual testing.

Reply:
Yes, this cleanup should be done in a separate PR.